vms/platformvm/txs/executor/camino_tx_executor_test.go

```diff
@@ -1541,7 +1534,7 @@ func TestCaminoRewardValidatorTx(t *testing.T) {
 		generateUTXOsAfterReward: func(txID ids.ID) []*avax.UTXO {
 			return []*avax.UTXO{
 				generateTestUTXO(txID, env.ctx.AVAXAssetID, defaultCaminoValidatorWeight, stakeOwners, ids.Empty, ids.Empty),
-				generateTestUTXOWithIndex(unlockedUTXOTxID, 2, env.ctx.AVAXAssetID, defaultCaminoBalance, stakeOwners, ids.Empty, ids.Empty, true),
+				generateTestUTXOWithIndex(unlockedUTXOTxID, 1, env.ctx.AVAXAssetID, defaultCaminoBalance, stakeOwners, ids.Empty, ids.Empty, true),
```

Comment:
Avalanche cortina 1 changes `defaultWeight` const that is used to build test genesis, this changes validators ordering, so test removes different validator than before. And this new validator has different utxo that we expect there. Everything is meticulously investigated, compared and tested on both new and old versions.